app-server/app/src/main/resources/logback.xml

```diff
@@ -0,0 +1,13 @@
+<configuration>
+    <appender name="file" class="ch.qos.logback.core.FileAppender">
+        <file>logging-app-server.log</file>
```

Comment:
Is there a way we can enable debug output based on environment (e.g. in development enable debug output, but in staging/prod disable debug output)?